eeps/eep-0065.md

```diff
@@ -0,0 +1,282 @@
+which the parser sorts out to be a record start.
+
+With simple prefix characters the scanner would have to be rewritten
+to recognize «`#bf`» as a new record token, a rewrite that might cause
+unexpected changes in record handling.  For example, today, «`# bf {`»
+is also a valid record start, so to be completely compatible the scanner
+would have to allow white-space or even newlines within the new record
+token, between «`#`» and the atom characters, which would be really ugly...
+
+For reasons, maybe some of them above, Elixir has chosen to use
+the «`~`» character as the start of a string prefix which they
+call a ["Sigil"][1].
+
+### Sigil
+
+In a general sense, a [Sigil][3], is a prefix to a variable
+that indicates its type, such as «`$I`» in Basic or Perl,
+ where «`$`» is the sigil and «`I`» is the variable.
+
+Here we define a Sigil as a prefix to a string literal
+that indicates how to interpret the string literal.
+It consists of:
+
+1. The character «`~`»
+2. An (optional) character in the range «`a`»-«`z`» or «`A`»-«`Z`»,
+   that specifies the [Sigil Type][].  The Sigil Type can
+   affect how the String Content is scanned, for example
+   if escape sequences are honoured.
+
+### String Delimiters
+
+Immediately following the [Sigil][] is the string start delimiter.
+A start delimiter has a corresponding end delimiter.
+
+The allowed delimiter start-end character pairs are:
+«`()`», «`[]`», «`{}`», and «`<>`».
+
+The following characters are both start and end delimiters:
+«`/`», «`|`», «`'`», and «`"`».  Triple-quote delimiters are also allowed,
+that is, a sequence of 3 or more «`"`» characters with newlines
+as described in [EEP 64][].
+
+### String Content
+
+Between the start and end [String Delimiters][], all characters
+are string content.  In a triple-quoted string all characters
+are verbatim.  In a string with single-character [String Delimiters][],
+normal Erlang escape sequences prefixed with «`\`» are honoured.
+The [Sigil Type][] can change this behaviour.
+
+Also, string interpolation should be performed for some [Sigil Type][]s.
+
+### Sigil Suffix
+
+After the String Content comes the end [delimiter][String Delimiters],
+and immediately after that the Sigil Suffix, that may be empty.
+
+The Sigil Suffix is a sequence of "name" characters, that is, those that
+are allowed as the second or later character in a variable or an atom.
+These are in short ISO [Latin-1][] letters, digits, «`_`» and «`@`».
+
+The Sigil Suffix may contain information about how to interpret
+the String Content, for a specific Sigil Type.  For example;
+regular expressions can make use of suffix flags to control
+expression compilation such as the «`i`» flag for a character case
+insensitive regular expression.
+
+### Sigil Type
+
+The suggested Sigil Types, inherited from Elixir, are:
+
+* «`s`»: binary string, an string represented as an UTF-8 encoded binary
+* «`c`»: char list, an Erlang string (Unicode codepoint list)
```

Comment:
Nitpick: the [types](https://www.erlang.org/doc/man/unicode#type-charlist) say `charlist` instead of `char_list`, so it may be worth making the spacing consistent here:

```suggestion
* «`c`»: charlist, an Erlang string (Unicode codepoint list)
```